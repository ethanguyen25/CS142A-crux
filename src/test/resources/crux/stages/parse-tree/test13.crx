// test returning int

int myInt() {
  return readInt();
}

void main() {
  printInt(myInt());
}
