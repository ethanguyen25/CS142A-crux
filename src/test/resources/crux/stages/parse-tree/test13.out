program
  declarationList
    declaration
      functionDefinition
        type
        parameterList
        statementBlock
          statementList
            statement
              returnStatement
                expression0
                  expression1
                    expression2
                      expression3
                        callExpression
                          expressionList
    declaration
      functionDefinition
        type
        parameterList
        statementBlock
          statementList
            statement
              callStatement
                callExpression
                  expressionList
                    expression0
                      expression1
                        expression2
                          expression3
                            callExpression
                              expressionList
